example.env

```diff
@@ -41,4 +43,17 @@ WEATHER_SRV_OPENWEATHERMAP_API_KEY=59b66f5817f09ae1da0f5e496f1ab132
 # Weather service MongoDB
 WEATHER_SRV_MONGO_INITDB_ROOT_USERNAME=root
 WEATHER_SRV_MONGO_INITDB_ROOT_PASSWORD=root
-WEATHER_SRV_MONGO_INITDB_DATABASE=openagridb
\ No newline at end of file
+WEATHER_SRV_MONGO_INITDB_DATABASE=openagridb
+
+# Pest and disease management
+PDM_ACCESS_TOKEN_EXPIRATION_TIME=240
+PDM_SERVICE_PORT=my_port
+PDM_BACKEND_CORS_ORIGINS=["*"]
```

Comment:
is this used?

Reply:
yeah, just added to yml, for some reason was missing, possibly when I was copying it over from the service_in_use to available_services dirs